desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,40 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### Security
+
+- Update Docker Engine to v20.10.14 to address [CVE-2022-24769](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-24769)
+- Update containerd to v1.5.11 to address [CVE-2022-24769](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-24769)
+
+### New
+
+- Add helper command and [instructions](https://docs.docker.com/desktop/mac/install/#install-from-the-command-line) for IT to install Docker Desktop remotely
+- Add [Docker SBOM plugin v0.6.0](https://github.com/docker/sbom-cli-plugin/releases/tag/v0.6.0)
+
+### Upgrades
+
+- [Docker Engine v20.10.13](https://docs.docker.com/engine/release-notes/#201013)
+- [Compose v2.4.1](https://github.com/docker/compose/releases/tag/v2.4.1)
+- [Buildx 0.8.2](https://github.com/docker/buildx/releases/tag/v0.8.2)
+- [containerd v1.5.11](https://github.com/containerd/containerd/releases/tag/v1.5.11)
+- [Go 1.18](https://golang.org/doc/go1.18)
+
+### Bug fixes and minor changes
+ - Deprecate macOS 10.14.
+ - Use `cri-dockerd` version 0.2.0 for new Kubernetes clusters.
+ - Fixed issue related to mysterious blank screen. Fixes [docker/for-mac#6134](https://github.com/docker/for-mac/issues/6134).
```

Comment:
Delete "mysterious"

Reply:
How about "Fixed an issue that sometimes caused Docker Desktop to display a blank white screen."